packages/framework/presence/src/baseTypes.ts

```diff
@@ -0,0 +1,11 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * A Fluid client connection identifier.
```

Comment:
Any more information we can provide here? What constitutes a valid identifier? How would a user acquire a valid one?

Reply:
We just say string in a lot of places. I wanted to at least name this. Ideally we'll brand it to distinguish from other strings. That is a bit hard at the moment since we just use `string` elsewhere. Adding remarks and private remarks.